app/views/user/shares/_modal_form.html.erb

```diff
@@ -1,9 +1,9 @@
 <div class="modal fade" id="modalSharesForm-<%= record.id %>" tabindex="-1" role="dialog" aria-labelledby="myModalLabel"
   aria-hidden="true">
-  <div class="modal-dialog" role="document">
+  <div class="modal-dialog modal-lg" role="document">
     <div class="modal-content">
       <div class="modal-header text-center">
-        <h4 class="modal-title w-100">Share</h4>
+        <h4 class="font-weight-bold modal-title w-100">Share birth record of <%= record.full_name %></h4>
```

Comment:
This will have some conflicts with #175 since it's not always something with a full name